packages/vufindview/src/Helper/HeadScript.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Head script view helper (extended for VuFind's theme system)
+ *
+ * PHP version 7
```

Comment:
I'm currently going through old PRs and making sure that none of them introduce outdated "PHP version" comments into the dev branch. If we want to move forward with this work, we need to remember to update this and any other comments listing version 7.